src/functionalTest/resources/features/F-206 - Request Notice Of Change/common/verify-case-events/F-206_Verify_NoC_Request_Event_Data_Alice.td.json

```diff
@@ -0,0 +1,43 @@
+{
+  "_guid_": "F-206_Verify_NoC_Request_Event_Data_Alice",
+  "_extends_": "F-206_Verify_NoC_Request_Event_Data__Base",
+
+  "specs": [
+    "to get Case Events API returns a NoCRequest event in which the user ID is set to Alice's email address AND the proxied_by field set to the ID of the system user"
```

Comment:
what does proxied_by field mean? I thought it should call data-store-api directly with the user who has read access to the events.